lib/puppet/network/auth_config_parser.rb

```diff
@@ -49,23 +49,23 @@ def parse_rights
   def parse_right_directive(right, var, value, count)
     value.strip!
     case var
-    when "allow"
-      modify_right(right, :allow, value, "allowing %s access", count)
-    when "deny"
-      modify_right(right, :deny, value, "denying %s access", count)
-    when "allow_ip"
-      modify_right(right, :allow_ip, value, "allowing IP %s access", count)
-    when "deny_ip"
-      modify_right(right, :deny_ip, value, "denying IP %s access", count)
-    when "method"
-      modify_right(right, :restrict_method, value, "allowing 'method' %s", count)
-    when "environment"
-      modify_right(right, :restrict_environment, value, "adding environment %s", count)
+    when _("allow")
```

Comment:
Should these be translated? The fact that they are used in a case statement makes me think they might not be external facing?